<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSYouTubeDownloader от КонтинентСвободы.рф</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>data/1.ico</normaloff>data/1.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>531</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>data/Баннер.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="leVideoLink">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ссылка на видео:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Версии:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bGetVers">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить версии</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bDownload">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Скачать версию</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbVersions">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Название видео:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>пусто</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_bar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbAdaptive">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>361</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Адаптивные форматы (могут быть ошибки)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>544</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
